molSimplify/Scripts/inparse.py

```diff
@@ -483,7 +483,7 @@ def cleaninput(args):
                 args.keepHs[i] = checkTrue(s)
     # parse FF settings:
     # if no FF opt is requested, turn off
-    if args.ffoption[0].lower() == ('n' or 'no'):
```

Comment:
good catch 😭😂

Reply:
interestingly `('n' or 'no')` this resolves to 'n' because of short-circuit evaluation...

Reply:
Had to run that snippet in order to figure out what it actually evaluates to. 😄